.idea/jsLibraryMappings.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="JavaScriptLibraryMappings">
+    <file url="file://$PROJECT_DIR$/uber/static/lib/analytics/reg_live.js" libraries="{jquery}" />
+    <file url="http://localhost:4321/magfest/analytics/recent_regs_live" libraries="{jquery}" />
+    <file url="PROJECT" libraries="{jquery}" />
+  </component>
+</project>
+
```

Comment:
ignore, this is OK, I accidentally deleted this file in a previous merge. pycharm stuff.